src/store/courses/actionCreators.js

```diff
@@ -0,0 +1,26 @@
+import {
+	ADD_COURSE,
+	DELETE_COURSE,
+	SAVE_COURSES,
+	UPDATE_COURSE,
+} from './actionTypes';
+
+export const saveCourses = (courses) => ({
+	type: SAVE_COURSES,
+	payload: courses,
+});
+
+export const addCourse = (course) => ({
```

Comment:
and here you save the course (but almost the same as add)

Reply:
Got it